packages/kbn-i18n/package.json

```diff
@@ -3,7 +3,7 @@
   "browser": "./target/web/browser.js",
   "main": "./target/node/index.js",
   "types": "./target/types/index.d.ts",
-  "module": "./src/index.js",
+  "module": "./src/index.ts",
```

Comment:
__note:__ I don't think we'll see the support of typescript modules as package.json "module", so I'd just remove "module" unless you have a reason to keep it.